static_src/components/navbar.jsx

```diff
@@ -112,7 +112,7 @@ export class Nav extends React.Component {
             subList = (
               <ul className={ secondList }>
                 <li>
-                  <a href={ this.orgHref(org) }>{ org.name }</a>
+                  <a href={ this.orgHref(org) }>{ org.name } overview</a>
```

Comment:
previously "spaces" — but it's actually our rudimentary org page

**note:** this element should get the active class `sidenav-active-dashboard` any time its page is visible. I didn't know how to do this.